index.html

```diff
@@ -4372,6 +4372,154 @@ <h2>Revision History</h2>
         </li>
       </ul>
     </section>
+    <section class="appendix informative">
+      <h2>Understanding Proof Sets and Proof Chains</h2>
+      <p>
+Proof sets and chains allow multiple proofs to be carried in a VC, i.e., instead
+of a single <em>proof</em> included in the signed VC we can express multiple
+proofs in an array as shown in the previous examples. The elements of this array
+are members of <em>proof set</em> and, optionally, a <em>proof chain</em>.
+The purpose of this section is to explain the intended use of each of these entities
+and, in particular, their differing security properties. These differing security
+properties lead to differences in the processing in section <a href="#add-proof-set-chain"></a>.
+      </p>
+      <p>
+We are going to represent VCs and their proofs in a highly abbreviated manner so
+that the key security properties can be observed.
+      </p>
+      <p>
+We consider a scenario with three signatories: a CEO, a CFO, and a VP of
+Engineering. Each will need to have a public and private key pair for signing a
+document. We denote the private/public keys of each of these signatories by
+<em>priv/pubCEO</em>, <em>priv/pubCFO</em>, and <em>priv/pubVPE</em>
```

Comment:
Is it really necessary to abbreviate `public` to `pub` for these? I've not abbreviated `secret` because I think `sec` implies too many possible expansions. I think it might be better to use `public` instead of `pub` throughout as well.
```suggestion
Engineering. Each will need to have a public key and secret key pair for signing a
document. We denote the secret/public keys of each of these signatories by
<em>secretCEO/pubCEO</em>, <em>secretCFO/pubCFO</em>, and <em>secretVPE/pubVPE</em>,
```